docs/ko/how-to_guides/ThanoSQL_connecting/rest_api_thanosql_tables.md

```diff
@@ -274,9 +274,9 @@ Use this method to execute the CREATE TABLE operation. In order to create the ta
     api_token = "Issued_API_TOKEN"
     table_name = "Table Name"
     base_url="https://{your-engine-url}/api/v1/table/"
-    schema_name = "Schema Name"
+    schema = "Schema Name"
 
-    # Note that the name and schema keys are missing from the body
+    # 본문에서 이름 및 스키마 키가 누락되었습니다.
```

Comment:
```suggestion
    # 본문(Body)에서 이름 및 스키마가 누락되었습니다.
```
What is a schema key?

Reply:
I'm not sure, the original text contains "key" so I just followed. But I think "key" should be removed!